docs/kyma-environment-broker/08-01-provisioning-kyma-environment.md

```diff
@@ -27,8 +27,17 @@ This tutorial shows how to provision Kyma Runtime on Azure using Kyma Environmen
    ```bash
    export AUTHORIZATION_HEADER="Authorization: Bearer $ACCESS_TOKEN"
    ```
-
-3. Make a call to the Kyma Environment Broker to create a Runtime on Azure. Find the list of possible request parameters [here](03-01-service-description.md).
+Alternatively, you can perform `kubectl port-forward` on the chosen pod to expose it on your local machine. Expose it to port `8080`:
```

Comment:
```suggestion
Alternatively, you can perform `kubectl port-forward` on the chosen Pod to expose it on your local machine. Expose it on port `8080`:
```
- According to our [guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#camelcase), we capitalized `Pod`. 
- `To` or `on` port? Can you check with a dev?